src/main/java/org/opentripplanner/raptor/api/request/PassThroughPoint.java

```diff
@@ -13,13 +14,13 @@ public class PassThroughPoint {
   private final int[] stops;
   private final String name;
 
-  public PassThroughPoint(int[] stops, final String name) {
+  public PassThroughPoint(final int[] stops, @Nullable final String name) {
```

Comment:
There is not point in adding `final` in method argument lists any more. We used to do it, but it make the reading of the signature a bit more "crowed" with very little gain. Having final fields in a class is something else, and using it in methods also can be helpful, but dividing the method into smaller method is usually better. This is a minor thing, so I want ask you to change it.

Reply:
Yes, I can agree on that.